follower-read.md

```diff
@@ -23,14 +23,14 @@ To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_re
 {{< copyable "sql" >}}
```

Comment:
@tier-cap please update line 21 from "To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` session variable:" to "To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` variable:"